routes/base.py

```diff
@@ -63,6 +63,10 @@ def contentful():
         session.get(
             'preview_token',
             environ.get('CONTENTFUL_PREVIEW_TOKEN', None)
+        ),
+        session.get(
+            'host',
+            environ.get('CONTENTFUL_HOST', None)
```

Comment:
Is this ever saved in session? I think this could simplified to `environ.get('CONTENTFUL_HOST', None)` without checking session data

Reply:
True! Fixed!